rpcs3/evdev_joystick_handler.cpp

```diff
@@ -251,98 +373,117 @@ std::vector<std::string> evdev_joystick_handler::ListDevices()
     return evdev_joystick_list;
 }
 
+int evdev_joystick_handler::add_device(const std::string& device, std::shared_ptr<Pad> pad)
+{
+	// Now we need to find the device with the same name, and make sure not to grab any duplicates.
+	fs::dir devdir{ "/dev/input/" };
+	fs::dir_entry et;
+	while (devdir.read(et))
+	{
+		// Check if the entry starts with event (a 5-letter word)
+		if (et.name.size() > 5 && et.name.compare(0, 5, "event") == 0)
+		{
+			std::string path = "";
```

Comment:
Is it normal to have an empty value for the variable `path` considering its (unique) usage at the line 408?
`auto it = std::find_if(devices.begin(), devices.end(), [path](EvdevDevice device) { return path == device.path; });`